stdlib/database/mysql/src/main/ballerina/src/mysql/client.bal

```diff
@@ -82,7 +113,7 @@ public type Options record {|
 |};
 
 # Possible options for SSL Mode.
-public type SSLMode "PREFERRED" | "REQUIRED" | "VERIFY_CERT" | "VERIFY_IDENTITY";
+public type SSLMode "PREFERRED"|"REQUIRED"|"VERIFY_CERT"|"VERIFY_IDENTITY";
```

Comment:
Shouldn't these be defined as separate types and then as a single union type? 
```
public const SSL_PREFERRED = "PREFERRED";
public const SSL_REQUIRED = "REQUIRED";
...

public type SSLMode SSL_PREFERRED|SSL_REQUIRED ..;
```

Then usage will be 
```
SSLMode mode = sql:SSL_PREFERRED;
```